firebase-middlewares.md

```diff
@@ -0,0 +1,58 @@
+## Using firebase middlewares
+
+In the Front-End, Where ever you are making a Http requests/fetch request like get,post and so on..., the headers need to be attached to these request and its done as below
+
+...
+import {getTokenWithHeaders} from '../../firebase/getTokenWithHeaders'
+
+const header = await getTokenWithHeaders()
+
+await fetch('http://localhost:5000/api/example-end-point', {
+method: "GET/POST",
+headers: header
+})
+.then(res => res.json())
+.then(data => console.log(data))
+.catch(err => console.log(err))
+...
```

Comment:
```javascript
import {getTokenWithHeaders} from '../../firebase/getTokenWithHeaders'

const header = await getTokenWithHeaders()

await fetch('http://localhost:5000/api/example-end-point', {
  method: "GET/POST",
  headers: header
})
.then(res => res.json())
.then(data => console.log(data))
.catch(err => console.log(err))
```

☝️ When you write code in a markdown file `.md` it makes it easier to read if you use the formatting feature 👍 
Like this:

![image](https://user-images.githubusercontent.com/16811871/78774695-50bef000-7995-11ea-92ff-ce078fc7aa16.png)

Reply:
Actually, you should be able to just insert an empty line and then indent your code with 4 spaces and it should be formatted as code.

    const example = () => "Like this";

Reply:
changed :)